lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss

```
@@ -309,12 +309,5 @@
       }
     }
   }
-
-  // @TODO: remove once radio button validation is properly added to Neue
-  .error {
-    .label {
-      display: none;
```

Going to sidestep that by just pre-selecting one of the options on initial load. The problem with hiding the error labels is that they were disappearing on the address form above as well, so users couldn't see errors when they were happening.